### PR TITLE
UI Tweaks.

### PR DESCRIPTION
**Pull request by parc-jason**
Show errors on Preview Merge popup.
Only show Get User Error if app thinks still logged in.
Remove spaces when parsing csv email addresses.
Force Jenkins to always build frontend.

# Description

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-z208i4-tools.pathfinder.gov.bc.ca/dashboard?id=nr-messaging-service-showcase-pr-71

File: Jenkinsfile [hunk 1]
```diff
@@ -730,7 +730,7 @@ def frontendBuilderTemplateUpdated() {
 }
 
 def frontendUpdated() {
```
Comment (loneil) [inline Jenkinsfile]: Are these fields necessary if they're just returning true?

File: frontend/src/ches/MergeForm.jsx [hunk 2]
```diff
@@ -711,6 +732,8 @@ class MergeForm extends Component {
 
     const senderPlaceholder = this.state.hasSenderEditor ? 'you@example.com' : this.state.config.sender;
 
+    const previewBodyDisplay = (this.state.apiValidationErrors && this.state.apiValidationErrors.length > 0) || (this.state.error && this.state.error.length >0) ? {display: 'none'} : {};
```
Comment (loneil) [inline frontend/src/ches/MergeForm.jsx]: Can multiline longer ternary operators for readability.

File: Jenkinsfile (hunk 1 above)
Comment (parc-jason) [reply Jenkinsfile]: I may re-instate that functionality.  no idea why the grep pattern started failing...

File: frontend/src/ches/MergeForm.jsx (hunk 2 above)
Comment (parc-jason) [reply frontend/src/ches/MergeForm.jsx]: will do on the next PR.

File: Jenkinsfile (hunk 1 above)
Comment (loneil) [reply Jenkinsfile]: kk

**Pull request merged**